src/solvers/flattening/boolbv_bitwise.cpp

```diff
@@ -64,13 +55,11 @@ bvt boolbvt::convert_bitwise(const exprt &expr)
           else if(expr.id()==ID_bitxnor)
             bv[i]=prop.lequal(bv[i], op[i]);
           else
-            throw "unexpected operand";
+            UNREACHABLE;
```

Comment:
I'd go with UNIMPLEMENTED or UNSUPPORTED or whatever happened to the legacy annotations.